rpcs3/Emu/System.cpp

```diff
@@ -987,7 +991,7 @@ void Emulator::Load(const std::string& title_id, bool add_only, bool force_globa
 		const std::string hdd0_disc = vfs::get("/dev_hdd0/disc/");
 		const std::size_t game_dir_size = 8; // size of PS3_GAME and PS3_GMXX
 		const std::size_t bdvd_pos = m_cat == "DG" && bdvd_dir.empty() && disc.empty() ? elf_dir.rfind("/USRDIR") - game_dir_size : 0;
-		const bool from_hdd0_game = m_path.find(hdd0_game) != umax;
+		const bool from_hdd0_game = m_path.find(hdd0_game) != std::string::npos;
```

Comment:
What's the problem with umax?

Reply:
why use umax if you can use the standard value that's proposed in official examples?

Reply:
It's too long

Reply:
to me this is easier to understand than some umax on a string find